ra/ra.go

```diff
@@ -59,6 +59,7 @@ type caaChecker interface {
 // NOTE: All of the fields in RegistrationAuthorityImpl need to be
 // populated, or there is a risk of panic.
 type RegistrationAuthorityImpl struct {
+	rapb.UnimplementedRegistrationAuthorityServer
```

Comment:
I don't think this one is accurate (yet). It will be necessary when the wrapper is removed, but for now I think this increases confusion about which struct actually implements the interface required by gRPC.